Gemfile

```diff
@@ -135,7 +135,7 @@ gem 'newrelic_rpm', '~> 3.10.0.279', group: [:staging, :production] # perf/error
 
 gem 'redcarpet', '~> 3.3.4'
 
-gem 'geocoder'
+gem 'geocoder', github: 'wjordan/geocoder', ref: 'rack-request-fix'
```

Comment:
We may want to merge in https://github.com/code-dot-org/code-dot-org/pull/10140 separately from this just for separate merge monitoring/rollback if anything comes up. This only affected ./bin/dashboard-server & caused JS and server log errors on the frontpage when running in that mode. Possibly only cosmetic.